youtube_dl/extractor/spankbang.py

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+
+class SpankBangIE(InfoExtractor):
+    """Extractor for http://spankbang.com"""
+    
+    _VALID_URL = r"https?://(?:www\.)?spankbang\.com/(?P<id>\w+)/video/.*"
+
```

Comment:
Just add a test video as described in [developer instructions](https://github.com/rg3/youtube-dl#developer-instructions) or look at any other extractor.